packages/react/src/Breadcrumb/index.ts

```diff
@@ -0,0 +1 @@
+export { Breadcrumb } from './Breadcrumb'
```

Comment:
Do we want to export types here as well?

Reply:
Not sure if we want or need to export the types. I guess something we might want to discuss. For now I will keep it consistent, so I exported it.